presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -61,7 +67,7 @@
     private final List<FilterFunction> filterFunctions;
     private final Map<Integer, Integer> filterFunctionInputMapping;   // channel-to-index-into-hiveColumnIndices-array mapping
     private final Set<Integer> filterFunctionInputs;                  // channels
-    private final Set<Integer> columnsWithFilters;                    // elements are indices into hiveColumnIndices array
+    private final Map<Integer, Integer> columnsWithFiltersScore;      // keys are are indices into hiveColumnIndices array; values are filter scores
```

Comment:
- typo: "keys are are" -> "keys are"
- columnsWithFiltersScore -> should it be `columnsWithFilterScores`?

Reply:
Roger that.